packages/neon/neon_files/lib/utils/task.dart

```diff
@@ -0,0 +1,66 @@
+part of '../neon_files.dart';
+
+sealed class FileTask {
```

Comment:
```suggestion
sealed class FilesTask {
```

Because of the app name :grimacing: